CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+- Remove package sync paragraph in package-management doc since
+  it's not available for salt clients and traditional is no
+  longer supported (bsc#1221279)
```

Comment:
```suggestion
- Remove package sync paragraph in package-management doc since
  it is not available for Salt clients and traditional clients are no
  longer supported (bsc#1221279)
```